pkgs/development/libraries/rocm-thunk/default.nix

```diff
@@ -31,6 +32,7 @@ stdenv.mkDerivation (finalAttrs: {
     libdrm
     numactl
     valgrind
+    gcc.cc.libgcc # TODO: unhack this?
```

Comment:
Oh, I'm sorry.  Some people run into this unexpected strictness (evaluating even on platforms that are unsupported/broken for the package).  Hopefully it will be OK now.

Reply:
Please try https://github.com/NixOS/nixpkgs/pull/225846 it is the proper fix for this.